### PR TITLE
HashCode Data in All Language

### PR DESCRIPTION
**Pull request by Sujith-Barve**

### CODE REVIEW

File: locale/Bengali.json [hunk 1]
```diff
@@ -94,6 +94,13 @@
       "searchHint": "অনুসন্ধান করুন",
       "continueLabel": "নির্বাচন করুন এবং চালিয়ে যান"
     },
+    "hashMarkInformation": {
+      "CreatedBy": "দ্বারা সৃষ্টি",
+      "IssuedBy": "প্রদান করেছেন",
+      "CreatedAt": "তৈরি",
+      "HashLabel": "হ্যাশ",
+      "Transaction": "লেনদেন"
```
Comment (amarts) [inline locale/Bengali.json:102]: @sankarshanmukhopadhyay @priyashankarbanerjee is this OK here?

File: locale/Hindi.json [hunk 2]
```diff
@@ -94,6 +94,13 @@
       "searchHint": "खोज",
       "continueLabel": "का चयन करें और जारी रखें"
     },
+    "hashMarkInformation": {
+      "CreatedBy":"के द्वारा बनाई गई",
+      "IssuedBy": "द्वारा जारी",
+      "CreatedAt": "को निर्मित",
+      "HashLabel": "हश",
+      "Transaction": "लेन-देन"
```
Comment (amarts) [inline locale/Hindi.json:102]: @sreepriyasree @sankarshanmukhopadhyay Please check..

File: locale/Kannada.json [hunk 3]
```diff
@@ -94,6 +94,13 @@
       "searchHint": "ಹುಡುಕಿ",
       "continueLabel": "ಆಯ್ಕೆಮಾಡಿ ಮತ್ತು ಮುಂದುವರಿಸಿ"
     },
+    "hashMarkInformation": {
+      "CreatedBy": "ರಚಿಸಿದವರು",
+      "IssuedBy": "ಇವರಿಂದ ನೀಡಲಾಗಿದೆ",
+      "CreatedAt": "ಸೃಷ್ಟಿಯ ದಿನಾಂಕ",
+      "HashLabel": "ಹ್ಯಾಶ್",
+      "Transaction": "ವ್ಯವಹಾರ"
```
Comment (amarts) [inline locale/Kannada.json:102]: @mankuthimma @aravindavk ?

File: locale/Malayalam.json [hunk 4]
```diff
@@ -94,6 +94,13 @@
       "searchHint": "തിരയുക",
       "continueLabel": "തിരഞ്ഞെടുത്ത് തുടരുക"
     },
+    "hashMarkInformation": {
+      "CreatedBy": "ഉണ്ടാക്കിയത്",
+      "IssuedBy": "പുറപ്പെടുവിച്ചത്",
+      "CreatedAt": "സൃഷ്ടിച്ച തീയതി",
+      "HashLabel": "ഹാഷ്",
+      "Transaction": "ഇടപാട്"
+    },
```
Comment (amarts) [inline locale/Malayalam.json:103]: @sreepriyasree @smohan-dw you are ok with this?

File: locale/Bengali.json (hunk 1 above)
Comment (Sujith-Barve) [reply locale/Bengali.json:102]: Need Check For Malayalam Also

Comment (sankarshanmukhopadhyay) [reply locale/Bengali.json:102]: I think I can get this improved today.

Comment (sankarshanmukhopadhyay) [reply locale/Bengali.json:102]: "CreatedAt": "সময়ে নির্মিত",

a single string change.

File: locale/Malayalam.json (hunk 4 above)
Comment (sreepriyasree) [reply locale/Malayalam.json:103]: no changes required

**Pull request merged**